dsa-solutions/lc-solutions/0500-0599/0540-single-element-in-sorted-array.md

```diff
@@ -0,0 +1,184 @@
+---
+id: single-element-in-sorted-array
+title: Single Element in a Sorted Array
+sidebar_label: Single Element in Sorted Array
+tags: 
+    - Binary Search
+    - Algorithm
+    - O(log n)
+---
+
+## Problem Description
+
+| Problem Statement                                       | Solution Link                                                              | LeetCode Profile                                        |
+| :------------------------------------------------------ | :------------------------------------------------------------------------- | :------------------------------------------------------ |
+| [Single Element in a Sorted Array](https://leetcode.com/problems/Single Element in a Sorted Array/description/) | [Single Element in a Sorted Array Solution on LeetCode](https://leetcode.com/problems/Single Element in a Sorted Array/solutions/) | [Nikita Saini](https://leetcode.com/u/Saini_Nikita/) |
+
+## Problem Description
+
+You are given a sorted array consisting of only integers where every element appears exactly twice, except for one element which appears exactly once. Return the single element that appears only once.
+
+Your solution must run in O(log n) time and O(1) space.
+
+### Examples
+
+## Example 1:
+
+**Input:** `nums = [1,1,2,3,3,4,4,8,8]`
+**Output:** `2`
+
+## Example 2:
+
+**Input:** `nums = [3,3,7,7,10,11,11]`
+**Output:** `10`
+
+### Constraints
+
+- \(1 \leq \text{nums.length} \leq 10^5\)
+- \(0 \leq \text{nums[i]} \leq 10^5\)
```

Comment:
replace `\(0 \leq \text{nums[i]} \leq 10^5\)` to `$0 \leq \text{nums[i]} \leq 10^5$`